src/test/scala/uk/gov/hmrc/ui/specs/DashboardSpec.scala

```diff
@@ -44,22 +48,35 @@ class DashboardSpec extends BaseSpec with StubPage {
 
       Then("The ratepayer is taken to the 'Check your answers' page")
       CheckYourAnswer.checkYourAnswer()
+      contactName = getElementByCssSelector("#contact-name-id").toString
       click(continueButton)
 
       Then("Ratepayer is taken to the Registration complete page")
       RegisterComplete.RegisterComplete()
       click(continueButton)
 
       Then("Ratepayer is now fully registered and is taken to the dashboard")
-      DashboardHome.DashboardHome()
+      DashboardHome.DashboardHome(contactName)
     }
 
     Scenario("Ratepayer is already registered and lands on the dashboard after login") {
 
       Given("Ratepayer logins through one login")
       loginOl()
       Then("Ratepayer is taken to the dashboard")
-      DashboardHome.DashboardHome()
+      DashboardHome.DashboardHome(contactName)
+
+    }
+
+    Scenario("Ratepayer lands on the dashboard and clicks the feedback link") {
+
+      Given("Ratepayer logins through one login")
+      loginOl()
+      Then("Ratepayer is taken to the dashboard")
+      DashboardHome.DashboardHome(contactName)
+      Then("Ratepayer clicks the feedback link and is taken to the feedback page")
+      feedbackLinkDisplay()
+      clickLink("feedback")
```

Comment:
After clicking on feedback link, just verify user is navigating to fedback page, just verify we are landing on feedback page.